### PR TITLE
update validation epoch

### LINKED ISSUE #7122: download_url doesn't automatically resolve fuzzy url
```
python /home/runner/work/model-zoo/model-zoo/ci/verify_bundle.py -b renalStructures_CECT_segmentation -m min

 	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses. 
start verifying renalStructures_CECT_segmentation:

You may still be able to access the file from the browser:

	 https://drive.google.com/uc?id=1Hm9293ypfWk9-uWHlxuolyxVA_FxfkVQ 

Access denied with the following error:
Traceback (most recent call last):
  File "/home/runner/work/model-zoo/model-zoo/ci/verify_bundle.py", line 133, in verify_bundle_directory
    download_large_files(bundle_path=bundle_path, large_file_name=large_file_name)
  File "/home/runner/work/model-zoo/model-zoo/ci/utils.py", line 108, in download_large_files
    download_url(**lf_data)
  File "/home/runner/.local/lib/python3.10/site-packages/monai/apps/utils.py", line 221, in download_url
    raise RuntimeError(
RuntimeError: Download of file from https://drive.google.com/file/d/1Hm9293ypfWk9-uWHlxuolyxVA_FxfkVQ/view?usp=sharing to models/renalStructures_CECT_segmentation/models/model.pt failed due to network issue or denied permission.

```

url in the format `https://drive.google.com/file/d/1Hm9293ypfWk9-uWHlxuolyxVA_FxfkVQ/view?usp=sharing` is not supported unless `fuzzy=True` is provided to `download_url`

### PR DESCRIPTION
**Pull request by wyli**
- this allows for validation epoch at the very beginning of training
- fixes #7122

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by KumoLiu — APPROVED** [review]

File: monai/engines/evaluator.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def run(self, global_epoch: int = 1) -> None:  # type: ignore[override]
 
         """
         # init env value for current validation process
-        self.state.max_epochs = global_epoch
+        self.state.max_epochs = max(global_epoch, 1)  # at least one epoch of validation
         self.state.epoch = global_epoch - 1
```
Comment (Nic-Ma) [inline monai/engines/evaluator.py:146]: For validation at start, so here `self.state.max_epochs` will be 1, `self.state.epoch` will -1? Will it run 2 epochs?

Comment (wyli) [reply monai/engines/evaluator.py:146]: it'll not run for 2 epochs according to the stopping condition https://github.com/pytorch/ignite/blob/854b5ea91629aefc5ab6fc35b5990190e3b35178/ignite/engine/engine.py#L733-L741

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**